content/docs/guides/contribute/contribute-code/frontend-conventions.en.md

```diff
@@ -59,9 +59,12 @@ Some additional conventions:
 
 ### Store/Redux
 
-Everything that is _selector_ is managed by the **view** and passed as props to components and sub-components.
+The store allows you to store data that will be accessible anywhere in the application. It is divided into slices, which correspond to our applications.
 
-Consequently, read and write calls to the store must be made at view level, irrigating the components proposed by the view with _props_ and _states_.
+However, we must be careful that our store does not become a catch-all.
+Adding new properties to the store must be justified by the fact that the data in question is “high-level” and will need to be accessible from locations far from the code base, and that a simple state or context variable is not relevant for storing this information.
```

Comment:
"far from the code base" is a bit blurry to me: the whole frontend code is the codebase. Maybe "far from each other" would be more precise?